CHANGELOG.md

```diff
@@ -35,13 +35,21 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
-### Breaking Changes
+
+### State Machine Breaking Changes
+
+
+### API Breaking Changes
```

Comment:
As mentioned in the Gaia PR, I don't understand the difference between API and Client breaking changes. Why not just have one? Here is how I see breaking changes:

1. State Machine Breaking
2. Client Breaking
  2a. REST Breaking
  2b. CLI Breaking
  2c. Everything else (e.g. Coins API)

Reply:
Ok, now I think I understand after reading your proposal. Let's update the changelog comments above to reflect the new stanzas.

Reply:
Just for anyone else reading this:

To me, `Client Breaking` is changes that will affect end-users of a blockchain. So any breaking changes to CLI commands/REST routes that would be interacted with by users, explorers, etc. go here.

`API Breaking` is changes that will affect developers building on top of the SDK. For example, a breaking change in the `Coins` API. This will not be relevant (and in fact will be invisible) to end-users like a block explorer. But it will be highly relevant for developers who may need to update their own software to upgrade to the latest SDK version.

Since the group primarily interested in `API Breaking Changes` (developers) and the group primarily interested in `Client Breaking Changes` (users/explorers/exchanges) are fairly disjoint, I have these sections separate and distinct rather than as subsections `2a, 2b, 2c`.